docs/cloudexplorer-with-minio.md

```diff
@@ -0,0 +1,61 @@
+# How to use Cloud Explorer with Minio [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+In this recipe you will learn how to carry out basic operations on Minio using Cloud Explorer.Cloud Explorer is a open-source S3 client. It works on Windows, Linux, and Mac. It has a graphical and command line interface for each supported operating system. If you have a feature suggestion or find a bug, please open an issue.
+
+# Features
+
+* Search
+* Performance testing
+* Music player
+* Transition buckets to Amazon Glacier  
+* Amazon RRS and Infrequently Accessed storage tiers
+* Migrate buckets between S3 accounts
+* Compress files prior to upload
+* Take screen shots to S3
+* Simple text editor
+* IRC client
+* Share buckets with users
+* Access shared buckets
+* View images
+* Sync folders
+* Graph CSV files and save them to a bucket
+* Record audio messages and save them to a bucket
+* Create snapshots of buckets
+
```

Comment:
Instead of the list of features, wouldn't it be better to mention couple of appealing features and a link to explore further here? Even better if you could put it from a Minio user perspective - why would they want to use this. (Something inline with [this](https://github.com/minio/cookbook/blob/master/CONTRIBUTING.md#advantages--use-case))

IMHO, the list makes the recipe look unnecessarily long.